### PR TITLE
release: v0.1.1, a couple of critical bug fixes

### PR DESCRIPTION
**Pull request by dgonyeo**
A couple of important bug fixes were merged in today, so I'd like to make a new minor release to include these.

### CODE REVIEW

**Comment by dgonyeo** [comment]
Actually I want to add a readme to the examples directory, so wait for that PR before merging this.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## v0.1.1
+
+appc spec version: v0.7.1+git5a7af19
+
+v0.1.1 has a couple of bug fixes since the initial release:
+
+- When the `--debug` flag was combined with the `run` command acbuild would
+  panic.
```
Comment (jonboulle) [inline CHANGELOG.md]: link to GH issue numbers

**Comment by jonboulle** [comment]
lgtm

**Comment by dgonyeo** [comment]
Wasn't an issue for one of them. That look ok?

**Comment by jonboulle** [comment]
That's fine (in rkt we just list the #xx number, but that's kinda a bug since gh doesn't autolink them then)

**Pull request merged**